packages/sanes-chrome-extension/src/utils/test/drawer.ts

```diff
@@ -0,0 +1,33 @@
+import { DRAWER_HTML_ID } from 'medulas-react-components/lib/components/Drawer';
+import TestUtils from 'react-dom/test-utils';
+import { click } from './dom';
+import { findRenderedDOMComponentWithId } from './reactElemFinder';
+
+let recoveryPhraseLink: Element;
+let requestsLink: Element;
+
+const initDrawer = async (dom: React.Component): Promise<void> => {
```

Comment:
I'd change two things here:
1. instead of `dom: React.Component` I would say: `drawerComponent: React.Component`.
2. The location of this file should be in `src/routes/account/test/` because we use the drawer only in this place.

Reply:
I don't understand the first one, the parameter is the whole dom of the view.